### DISCUSSION TITLE
validation set

### DISCUSSION THREAD

**Discussion by J-You** [deepmodeling/deepmd-kit#849] [Q&A] [answered]
Hi dpmd developers,

I have some problems while using the v2.0.0-beta.3:

1. `sel` key seems only can assign a list, while for a str, i.e., `auto` or `auto:factor` can not be read.

2. For a certain atomic type, if its coordination number is different in the multi-systems, how could I set `sel`? Maybe choose the larger one?

3. How do I construct a reasonable validation set?  For example, if my system contains three temperatures as 1000K, 900K, and 800K. I am not sure which of the following is more appropriate:
Pick up the frames from three temperatures randomly, regarding other frames as training set
Just set the frames of 800K as validation set, regarding 1000K and 900K data as training set

Thanks for your time and attention.

**Comment by njzjz** [comment, marked as answer]
1. The feature is newer than v2.0.0.b3.
2. sel is used by the model, and we only have one model no matter how many data sets you have.
3. The validation set should keep consistent with the training set.

**Reply by J-You** [reply]
Thanks for your reply. I have got it.
